lib/db_connect.js

```diff
@@ -28,7 +31,14 @@ module.exports = function () {
           function (err) {
             if (err && err.message && err.message.indexOf('ECONNREFUSED') > -1) {
               log.warn('db', {message: 'Failed to connect to database, retrying...'})
-              return P.delay(dbConnectionRetry).then(dbConnectPoll)
+              if (!cancelled) {
+                return P.delay(dbConnectionRetry)
+                  .then(() => {
```

Comment:
@seanmonstar i think l10n extract cannot parse this, so you either need to change this to use `function` or update to  l10n modules. I suggest just removing the `() => {`  and we can update the l10n modules later....